hw3/03-form.js

```diff
@@ -0,0 +1,29 @@
+const http = require('http');
+const express = require('express');
+var bodyParser = require('body-parser');
+const app = express();
+
+const port = process.env.PORT || 5001;
+
+// http://localhost:5001/form should return a form with input elements for username, email, and submit button
+
+// http://localhost:5001/submit should return all the data the user entered
+
+app.get('/', (req, res) => {
```

Comment:
I believe app.get is a functionality in Express JS. Can you check and confirm if we are allowed to use it in this assignment as I have submitted my code using Node JS only.